functests/4_latency/latency.go

```diff
@@ -428,6 +434,16 @@ func createLatencyTestPod(testPod *corev1.Pod, node *corev1.Node, logName string
 	err = pods.WaitForPhase(testPod, corev1.PodRunning, 2*time.Minute)
 	Expect(err).ToNot(HaveOccurred())
 
+	if runtime, _ := strconv.Atoi(latencyTestRuntime); runtime > 1 {
+		By("Checking actual CPUs number for the running pod")
+		limitsCpus := testPod.Spec.Containers[0].Resources.Limits.Cpu()
+		RequestsCpus := testPod.Spec.Containers[0].Resources.Requests.Cpu()
+		//letancy pod is guaranteed
+
+		Expect(fmt.Sprint(limitsCpus)).To(Equal(fmt.Sprint(latencyTestCpus)), fmt.Sprintf("actual limits of cpus number used for the latency pod is not as set in LATENCY_TEST_CPUS, actual number is: %s", limitsCpus))
```

Comment:
better to use the methods of a `resource.Quantity`, like `Cmp`

Reply:
Glad to know that. Modified it. Thanks!